lib/AST/ConformanceLookup.cpp

```diff
@@ -70,10 +70,46 @@ ModuleDecl::lookupExistentialConformance(Type type, ProtocolDecl *protocol) {
 
   assert(type->isExistentialType());
 
+  auto getConstraintType = [&type]() {
+    if (auto *existentialTy = type->getAs<ExistentialType>())
+      return existentialTy->getConstraintType();
+    return type;
+  };
+
+  auto lookupSuperclassConformance = [&](ExistentialLayout &layout) {
+    if (auto superclass = layout.explicitSuperclass) {
+      if (auto result =
+              lookupConformance(superclass, protocol, /*allowMissing=*/false)) {
+        if (protocol->isSpecificProtocol(KnownProtocolKind::Sendable) &&
```

Comment:
Is the true branch reachable? It seems we only call this if !protocol->existentialConformsToSelf() but that would be false for Sendable

Reply:
I just moved this code over so I'm not sure.

Reply:
I saw this in a couple other places I think, will investigate!

Reply:
Okay, we do call this in two places where `!protocol->existentialConformsToSelf()` and later on in case when it is, so it would be reachable at leave in one spot.